src/app/mauro/data-model.service.ts

```diff
@@ -227,6 +227,19 @@ export class DataModelService {
       .pipe(map((response: DataModelDetailResponse) => response.body));
   }
 
+  /**
+   * Moves a data model to a target folder.
+   *
+   * @param modelId The ID of the data model to move.
+   * @param targetFolderId The ID of the folder to move to.
+   * @returns An observable of {@link DataModelDetail} object just moved.
+   */
+  moveToFolder(modelId: Uuid, targetFolderId: Uuid): Observable<DataModelDetail> {
```

Comment:
Should this be called MoveDataModelToFolder to match the convention for the rest of the service?

Reply:
There is an `addToFolder()` function, so seems consistent to me to have a `moveToFolder()` function too.